test-result-summary-client/src/Build/BuildTable.jsx

```diff
@@ -33,10 +33,18 @@ export default class TestTable extends Component {
             if ( value.type === "Build" ) {
                 if ( value.hasChildren ) {
                     return <Link to={{ pathname: '/buildDetail', search: params( { parentId: value._id } ) }}
-                        style={{ color: resultColor }}> {value.buildName} </Link>;
+                        style={{ color: resultColor }}> {value.buildName} # {value.buildNum} </Link>;
                 } else {
                     return <Link to={{ pathname: '/output/build', search: params( { id: value._id } ) }}
-                        style={{ color: resultColor }}> {value.buildName} </Link>;
+                        style={{ color: resultColor }}> {value.buildName} # {value.buildNum} </Link>;
+                }
+            }else if ( value.type === "Perf" ) {
+                if ( value.hasChildren ) {
+                    return <Link to={{ pathname: '/buildDetail', search: params( { parentId: value._id } ) }}
+                        style={{ color: resultColor }}> {value.buildName} # {value.buildNum} </Link>;
+                } else {
+                    return <Link to={{ pathname: '/output/build', search: params( { id: value._id } ) }}
+                        style={{ color: resultColor }}> {value.buildName} # {value.buildNum} </Link>;

```

Comment:
There is no special treatment for Perf. Could we just remove the if else altogether?

Reply:
Code changed as required. Please check.